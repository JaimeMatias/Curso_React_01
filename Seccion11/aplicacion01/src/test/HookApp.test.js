import React from "react";
import { shallow } from 'enzyme';
import { HookApp } from "../HookApp";

describe('Prueba <HookApp />', () => {
    //Generar el Snapshot
    
    test('debe mostrar el Componente correctamente ', () => {
        const wrapper = shallow(<HookApp />);
        //Compara el componente generado con el snapshot que tenemos
        expect(wrapper).toMatchSnapshot();
    });

})
