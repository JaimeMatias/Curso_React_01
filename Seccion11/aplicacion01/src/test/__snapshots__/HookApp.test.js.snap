// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba <HookApp /> debe mostrar el Componente correctamente  1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
